tests/fragments/test_generator.py

```diff
@@ -42,6 +42,23 @@ def test_schema_generator(template_fragment, tmpdir):
     __validate_against_meta_schema(schema)
 
 
+def test_schema_generator_with_complex_real_module(template_fragment, tmpdir):
+    schema = __generate_schema("OpsWorks.json", template_fragment)
+    schema_file = tmpdir.join("schema.json")
+    assert os.path.exists(schema_file)
+    generated_schema = __read_file(schema_file)
+    expected_schema = __read_file(
+        os.path.join(directory, "../data/sample_fragments/OpsWorksSchema.json")
+    )
+    assert generated_schema == expected_schema
```

Comment:
According to [this answer](https://www.quora.com/How-do-I-compare-two-JSON-files-in-Python?share=1) for "How do I compare two JSON files in Python?"
> the == operator on two dicts doesn’t compare the ordering of the keys. 

Suggestion: If we find the JSON diff and print it as an error log here in case of failure, that'll help us understand what changed.

Reply:
These are not dicts. This is a simple string comparison.

Reply:
Sorry, I thin I know what you're saying. I think it's not necessary to make the test pass when the order in the json is different. There is no reason for that to happen without an intentional change to the schema generation.

Reply:
> I think it's not necessary to make the test pass when the order in the json is different.

Got it 👍